pkg/bucketeer/user/user.go

```diff
@@ -0,0 +1,21 @@
+package user
+
+// User is the Bucketeer user.
+//
+// User contains a mandatory user id and optional attributes (Data) for the user targeting and the analysis.
+
+type User struct {
+	ID   string            `json:"id,omitempty"`
+	Data map[string]string `json:"data,omitempty"`
+}
+
+// Valid returns true if valid user, otherwise returns false.
+func (u *User) RestValid() bool {
```

Comment:
Where is this being used?

Reply:
This will be replacement of https://github.com/ca-dp/bucketeer-go-server-sdk/blob/master/pkg/bucketeer/user.go#L25.